### PR TITLE
[web-task-split] Update rsyslog_configurer to use the built in Task functionality

### PR DESCRIPTION
**Pull request by jessicamack**
Signed-off-by: jessicamack <jmack@redhat.com>

##### ISSUE TYPE

 - Breaking Change 
 - New or Enhanced Feature
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - API
 - UI
 - Collection
 - CLI
 - Docs
 - Other

##### AWX VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by AlanCoding — APPROVED** [review]
I think this looks great!

File: awx/main/utils/external_logging.py [hunk 1]
```diff
@@ -115,6 +116,7 @@ def escape_quotes(x):
     return tmpl
 
 
+@task(queue='rsyslog_configurer')
```
Comment (relrod) [inline awx/main/utils/external_logging.py:119]: But then dispatcher also needs to know to listen on this queue, right? @AlanCoding

Comment (AlanCoding) [reply awx/main/utils/external_logging.py:119]: Yes, I wasn't sure if that was getting added here, or done separately. It is a corresponding change that needs to go with these changes.

**Review by djyasin — APPROVED** [review]

**Review by thedoubl3j — APPROVED** [review]

File: awx/main/utils/external_logging.py (hunk 1 above)
Comment (jessicamack) [reply awx/main/utils/external_logging.py:119]: Added the channel to the dispatcher queues

**Review by AlanCoding — APPROVED** [review]
looks like this should complete the work now

**Review by thedoubl3j — APPROVED** [review]

**Review by relrod — APPROVED** [review]

**Pull request merged**